test/short-read.c

```diff
@@ -78,6 +78,10 @@ int main(int argc, char *argv[])
 		return 1;
 	}
 
+	if (cqe->res == -EINVAL) {
```

Comment:
This one seems odd, readv has been supported since day 1?

Reply:
Right sorry, this one is failing for me with cqe->res == 0, and I think I might have checked for errno == EINVAL or similar and then this mutated into that after the fact and was left over. Removed now.